core/wrapper/activetimer.go

```diff
@@ -0,0 +1,68 @@
+package wrapper
```

Comment:
Der timer gehört eher in core denn in wrapper- er wrappt ja nix?

Reply:
hatte einfach den Chargetimer kopiert der auch dort ist. Hatte mich auch schon gefragt warum er wrapper ist.

Reply:
geändert in core